### ISSUE TITLE
Change to NWFSC Data Warehouse URL

### ISSUE THREAD

**Issue by Curt-Whitmire-NOAA** [James-Thorson/FishData#24]
Hi Jim,
NWFSC IT recently changed the server URL for the FRAM Data Warehouse. The new URL adds "webapps" to the front end of the URL, like the example below:

https://www.webapps.nwfsc.noaa.gov/data/api/v1/source/trawl.catch_fact

**Comment by afredston** [comment]
I assume this breaks `download_catch_rates()`? I can submit a PR to update the URLs if that's helpful...

**Comment by James-Thorson-NOAA** [comment]
Yeah, I haven't checked.  Are you interested in taking a moment to confirm and then do the PR @afredston ...?  Or tell me if you're too busy and ... err... I might ignore it until I'm back from leave :0

**Comment by afredston** [comment]
happy to do it this week! stay tuned

**Comment by Curt-Whitmire-NOAA** [comment]
Thanks @afredston! Yes, the new URL causes download_catch_rates() to not access data for survey == "WCGBTS", "WGHL" and "WGT".

**Comment by Curt-Whitmire-NOAA** [comment]
I should have wrote the "current URL" causes download_catch_rates() to not access data...

**Comment by kellijohnson-NOAA** [comment]
Perhaps another option would be to rely on nwfsc-assess/nwfscSurvey::PullCatch.fn that @chantelwetzel-noaa maintains.

**Comment by James-Thorson-NOAA** [comment]
@chantelwetzel-noaa @kellijohnson-NOAA Could either of you comment on what output nwfsc-assess/nwfscSurvey::PullCatch.fn(.) generates, or even how it compares with the function here?

I've always hoped to transition to each region maintaining their own function to pull their API, so that FishData is really more about harmonizing and publicizing those regional sources (as needed realistically for any comparative analysis).  So I'm excited to learn about that other option!

**Comment by kellijohnson-NOAA** [comment]
@James-Thorson-NOAA nwfscSurvey::PullCatch.fn(SurveyName = "NWFSC.Combo") results in a data frame with the following columns:
Trawl_id
Common_name
Project
Year
Pass
Vessel
Tow
Date
Depth_m
Longitude_dd
Latitude_dd
Area_Swept_ha
Subsample_count
Subsample_wt_kg
total_catch_numbers
total_catch_wt_kg
cpue_kg_km2

so I think we would have to add a few columns to match what you have, though I don't know exactly what columns are necessary.
